src/algorithms/meta/SubDivideFunctors.h

```diff
@@ -72,8 +87,8 @@ class GeometrySplit {
         }
     }
 
-    std::vector<std::vector<long int>> m_ids;
-    std::vector<std::string> m_divisions;
+     std::vector<std::vector<long int>> m_ids;
```

Comment:
Looks like this breaks indentation, not fix it. May be mobile code display issue.